dashboard/config/locales/en.yml

```diff
@@ -311,6 +311,8 @@ en:
       app_lab_tutorials: "App Lab Tutorials"
       game_lab_documentation: "Game Lab Documentation"
       game_lab_tutorials: "Game Lab Tutorials"
+      sprite_lab_documentation: "Sprite Lab Documentation"
```

Comment:
Can you add these to the [i18n gsheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit) as well (search for the corresponding game lab or app lab strings and add them around there). I don't actually think it's necessary but because the header code is shared, it'll be safer.

Reply:
Done! Could you give me a quick description (maybe Slack easier, or a link to a doc overview of the i18n pipeline?) on how this works?

Reply:
https://github.com/code-dot-org/code-dot-org/tree/staging/i18n is the best documentation we have. The gsheet holds the strings for the code.org site, which is likely overkill here but just wanted to be on the safe side.